.cirrus.yml

```
@@ -222,6 +253,35 @@ validate_task:
     always: *runner_stats
 
 
+validate_aarch64_task:
+    name: "Validate $DISTRO_NV aarch64 Build"
+    alias: validate_aarch64
+    # This task is primarily intended to catch human-errors early on, in a
+    # PR.  Skip it for branch-push, branch-create, and tag-push to improve
+    # automation reliability/speed in those contexts.  Any missed errors due
+    # to nonsequential PR merging practices, will be caught on a future PR,
+    # build or test task failures.
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: *is_pr
+    depends_on:
+        - ext_svc_check
+        - automation
+        - build_aarch64
+    # golangci-lint is a very, very hungry beast.
+    ec2_instance:
+        <<: *standard_build_ec2_aarch64
+        type: t4g.xlarge
+    env:
+        <<: *stdenvars
+        TEST_FLAVOR: validate
+        VM_IMAGE_NAME: ${FEDORA_AARCH64_AMI_ID}
+    # N/B: This script depends on ${DISTRO_NV} being defined for the task.
+    clone_script: *get_gosrc
```

```suggestion
-    clone_script: *get_gosrc
    # N/B: This script depends on ${DISTRO_NV} being defined for the task.
    clone_script: &get_gosrc_aarch64 |
        cd /tmp
        echo "$ARTCURL/build_aarch64/repo/repo.tbz"
        time $ARTCURL/build_aarch64/repo/repo.tbz
        time tar xjf /tmp/repo.tbz -C $GOSRC
```

That way you can use the alias `*get_gosrc_aarch64` in subsequent tasks